src/app/shared/services/blockchain/campaign.service.ts

```diff
@@ -269,6 +269,7 @@ interface UploadInfoData {
   startDate: iso8601
   endDate: iso8601
   return: {
+    frequency?: ReturnFrequency,
```

Comment:
can we find a better way for it? like... rate? period? I don't know. How do people call this in real finance stuff? 
If there is no better term, leave it like this.

Reply:
I think something like `annuityFrequency` would be more appropriate here. I am not 100% sure though, but that seems like a fitting name.

Reply:
Or maybe `annuityPeriod`, what do you think which one is better?

Reply:
ahh.. annuity refers to annual... doesn't matter, leave it like this :)